natgeo/mortar
# [SYTO-869] Pass in initial width prop from configurator

This PR is a follow up to https://github.com/natgeo/mortar/pull/264

https://jira.natgeo.com/browse/SYTO-869

To test:
- Pull SYTO-869
- Run `gulp test` to verify tests and linting pass
- Run `gulp serve`
- Verify component layout on initial load at http://localhost:3000/contentPackages/contentPackage/index.html at the various break points
